lntemp/harness.go

```diff
@@ -1565,10 +1565,17 @@ func (h *HarnessTest) MineEmptyBlocks(num int) []*wire.MsgBlock {
 // QueryChannelByChanPoint tries to find a channel matching the channel point
 // and asserts. It returns the channel found.
 func (h *HarnessTest) QueryChannelByChanPoint(hn *node.HarnessNode,
-	chanPoint *lnrpc.ChannelPoint) *lnrpc.Channel {
+	chanPoint *lnrpc.ChannelPoint, peerAliasLookup bool) *lnrpc.Channel {
```

Comment:
Why not just expect the options to be passed here? Then you don't need all the lines that add the `, false` to the `QueryChannelByChanPoint`?